src/java.base/share/classes/java/nio/file/SecureDirectoryStream.java

```diff
@@ -135,6 +135,182 @@ SeekableByteChannel newByteChannel(T path,
                                        FileAttribute<?>... attrs)
         throws IOException;
 
+    /**
+     * Creates a new and empty file, failing if the file already exists.
+     *
+     * <p>This method works in a similar manner to {@linkplain Files#createFile
```

Comment:
I think this method can use "This method works in exactly the manner specified".

Reply:
Fixed in b59c3f6.